client/src/res/values/joyride.js

```diff
@@ -50,18 +50,30 @@ export const JOYRIDE_STEPS = [
           target: '.leaflet-states-pane .leaflet-interactive:nth-child(4)',
           content: 'Click on California to unlock it and then click "Next"',
           spotlightClicks: true,
+          disableBeacon:true,
+        },// At the end of step 7, need to jump zoom to la county
+        { //8
+          target: '.leaflet-counties-pane .leaflet-interactive:nth-child(240)',
+          content: 'Incidents can also be viewed at a county level with the same functionality as states. Zoom in to see data for individual counties.',
+          spotlightClicks: true,
+          disableBeacon: true,
+        },
+        {// 9
+          target: '.MuiToggleButtonGroup-root',
+          content: 'Use this location switcher to jump to view incidents within Alaska and Hawaii',
```

Comment:
Are there points when we want to hide the back button? Certain states, the back button doesn't work as intended or causes weird effects

Reply:
Got it, makes sense for a lot of the steps that are dependent upon opening or clicking something. I hid the back button on all steps that it breaks the tutorial or simply doesn't go back because of the joyride callback blocking it.